mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -2570,12 +2581,20 @@ _PUBLIC_ enum MAPISTATUS EcDoRpc_RopSyncImportDeletes(TALLOC_CTX *mem_ctx,
 	owner = emsmdbp_get_owner(synccontext_object);
 	openchangedb_get_MailboxReplica(emsmdbp_ctx->oc_ctx, owner, &repl_id, &replica_guid);
 
+	object_array = &request->PropertyValues.lpProps[0].value.MVbin;
```

Comment:
Here, it applies even more than above. In this PR I just move the code a few lines earlier.

If we want to make sure any other change does not break things, I would suggest to put off the additional checks to other pull request. Indeed, we have to add mapitest and client side ImportDeletes operation to make sure everything works as expected :)